client/cli/src/params/network_params.rs

```diff
@@ -127,6 +127,10 @@ pub struct NetworkParams {
 	#[arg(long)]
 	pub kademlia_disjoint_query_paths: bool,
 
+	/// Kademlia replication factor.
```

Comment:
Could you give a brief explanation in the comment as to what this does?

Reply:
You mean in CLI arg comment or here?

Replication factor governs to how many other peers must value be sent to when inserting value into DHT for insertion to be successful, it also governs how many peers are queried when key lookup is being done.

Reply:
By default it's set to 20 which is the reason why AuthorityDiscovery failed to track peer Multiaddrs. For development and testing purposes we want to configure this value.